special/special.js

```diff
@@ -0,0 +1,64 @@
+//API variables
+var foodApiID = '3c019435';
+var foodApiKey = '989fd6474c5656909bd6106a3b238820';
+
+//get query selections from choice buttons
+var qword = ['breakfast','lunch','dinner','chicken','beef','pork','fish','tofu','american','caribbean','indian','asian','mexican','mediterranean'];
+var harray = ['vegan', 'vegetarian', 'sugar-conscious', 'peanut-free', 'tree-nut-free', 'alcohol-free'];
+var darray = ['balanced', 'high-protein', 'low-fat', 'low-carb'];
+
+//gets a random word from each variable
+var rqword = qword[Math.floor(Math.random()*qword.length)];
+var rharray = harray[Math.floor(Math.random()*harray.length)];
+var rdarray = darray[Math.floor(Math.random()*darray.length)];
+
+//Edamam recipie API call based on health and diet selections
+$.ajax({
+        url: 'https://api.edamam.com/search?q=' + rqword + '&health=' + rharray + '&diet=' + rdarray + '&app_id=' + foodApiID + '&app_key=' + foodApiKey,
```

Comment:
you probably want to construct this string over multiple lines so it's easier to pick out the individual parts should you ever need to modify it

```
'https://api.edamam.com/search'
     + ?q=' + rqword 
     + '&health=' + rharray 
     + '&diet=' + rdarray 
     + '&app_id=' + foodApiID 
     + '&app_key=' + foodApiKey
```